poly1305/src/backend/avx2.rs

```diff
@@ -152,6 +168,24 @@ impl State {
         };
         tag_int.write(tag.as_mut_slice());
 
-        Tag::new(tag)
+        tag
+    }
+}
+
+impl BlockSizeUser for State {
+    type BlockSize = U16;
+}
+
+impl ParBlocksSizeUser for State {
+    type ParBlocksSize = U64;
```

Comment:
Bug: this should be `U4`. I was misled by the associated type being named `ParBlocksSize`, which made me think it had the same units as `BlockSize` (i.e. bytes, not blocks).